exercises/allergies/example.js

```diff
@@ -15,7 +15,8 @@ class Allergies {
   }
 
   list() {
-    return possibleAllergies.filter((allergy, i) => this.allergenIndex & Math.pow(2, i));
+    // eslint-disable-next-line no-bitwise
```

Comment:
I suggest we disable this at project level. More detailed comment here: https://github.com/exercism/javascript/issues/480#issuecomment-432102095

Reply:
hear hear.

Reply:
Thanks for the feedback guys 🙏😌
I removed the directive comment as requested.